src/coreclr/nativeaot/Runtime/eventpipe/ds-rt-aot.cpp

```diff
@@ -238,8 +238,8 @@ uint32_t
 ds_rt_aot_set_environment_variable (const ep_char16_t *name, const ep_char16_t *value)
 {
 #ifdef TARGET_UNIX
-    ep_char8_t *nameNarrow = ep_rt_utf16le_to_utf8_string (name, ep_rt_utf16_string_len (name));
-    ep_char8_t *valueNarrow = ep_rt_utf16le_to_utf8_string (value, ep_rt_utf16_string_len (value));
+    ep_char8_t *nameNarrow = ep_rt_utf16le_to_utf8_string_n (name, ep_rt_utf16_string_len (name));
```

Comment:
Can we use the `ep_rt_utf16_to_utf8_string` overload here and other similar places that just call `ep_rt_utf16_string_len` for the second argument?